README.md

```diff
@@ -30,14 +30,14 @@ mentored by the [MacPorts](https://www.macports.org) organisation.
 ___
 
 # Running the App
-The repository contains Docker configuration which can be used out of the box, however, the app can be run without using
-the docker container. Both way are discussed below:
+The repository contains Docker configuration which can be used to build docker image, however, the app can be run without using
```

Comment:
```suggestion
The repository contains Docker configuration which can be used to build the docker image, however, the app can be run without using
```

Reply:
Thank you. Would you mind if I create a single commit with all these improvements?